components/Chips/tests/unit/ChipViewShapeThemerTests.m

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialChips+ShapeThemer.h"
+#import "MaterialChips.h"
```

Comment:
_Tip:_ 
```
#import "MaterialChips.h"
#import "MaterialChips+ShapeThemer.h"
```

Reply:
Unfortunately, clang-format would complain.  "+" comes before "." in ASCII sorting.

http://www.asciitable.com/

Reply:
😭